src/scripts/eos-analysis

```diff
@@ -249,6 +249,10 @@ The output file will be stored in EOS_BASE_DIRECTORY/POSTERIOR/nested.
         help = 'The maximum number of iterations. Iterations may stop earlier if the termination condition is reached.',
         dest = 'maxiter', action = 'store', type = int, default = None
     )
+    parser_sample_nested.add_argument('-s', '--rng-seed',
```

Comment:
Can you use the command `--use-random-seed` for consistency with the `find-mode` task?